.github/workflows/minimum_dependency_checker.yml

```diff
@@ -0,0 +1,77 @@
+name: Minimum Dependency Checker
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'requirements.txt'
+      - 'test-requirements.txt'
+      - 'core-requirements.txt'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Run min dep generator - test requirements
+        id: min_dep_gen_test
+        uses: alteryx/minimum-dependency-generator@v2
```

Comment:
Does this action always list the packages in the same order? I ask cause in #2268, I notice that the packages are not listed in alphabetical order and `diff` cares about line order

Reply:
yes, the packages are always in the same order. I can update `minimum-dependency-generator` to generate them in alphabetical order if you want (thought it shouldn't block this MR).

Reply:
No need, I was just wondering!